File: golang/pattern_1.go
/*
Problem Link: https://www.codingninjas.com/studio/problems/n-forest_6570177?utm_source=youtube&utm_medium=affiliate&utm_campaign=striver_patternproblems
*/

package main

import ("fmt")



func main(){
	printPattern(3)
}

func printPattern(N int){
	for i:=0; i<N; i++ {
		for j:=0; j<N; j++ {
			fmt.Print("*")
		}
		fmt.Println()
	}
}

File: rust/src/pattern_1.rs

/*
Problem Link: https://www.codingninjas.com/studio/problems/n-forest_6570177?utm_source=youtube&utm_medium=affiliate&utm_campaign=striver_patternproblems
*/

pub fn print_pattern(N: i32){
    for i in 1..=N {
        for j in 1..=N {
            print!("*");
        }
        println!();
    }
}